src/explore-education-statistics-admin/src/pages/release/data/ReleaseApiDataSetPreviewTokenPage.tsx

```diff
@@ -109,7 +134,18 @@ export default function ReleaseApiDataSetPreviewTokenPage() {
                   <p>Are you sure you want to revoke this token?</p>
                 </ModalConfirm>
                 <p>
-                  <Link to="todo">View API data set token log</Link>
+                  <Link
+                    to={generatePath<ReleaseDataSetRouteParams>(
+                      releaseApiDataSetPreviewTokenLogRoute.path,
+                      {
+                        publicationId,
+                        releaseId,
+                        dataSetId,
+                      },
+                    )}
```

Comment:
Could use the `tokenLogPagePath` variable here?

Reply:
yep, done now cheers.